src/main/resources/attackvectors/UnionBasedSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,7 @@
+UNION_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=By sending a query with <i>UNION</i> word we can gain data about users or other internal information from the databases. An example for this is:<br/>\
```

Comment:
I think the information about union is wrong. Please check the description about union query and add that.